src/components/notifications/utils.js

```diff
@@ -35,15 +33,3 @@ export function isViceNotification(notification) {
         message?.payload?.interactive_urls?.length > 0
     );
 }
-
-/*
- * Takes in a notification object and returns the time
- * stamp of the notification in 'pretty format'
- */
-export function getTimeStamp(timestamp) {
-    if (timestamp) {
-        // slicing because time has extra zeroes in the unix string
-        const d = fromUnixTime(timestamp.slice(0, -3));
-        return formatDistance(d, new Date());
-    }
-}
```

Comment:
Moved this to `components/utils/DateFormatter.js` file so that it can be used in `dashboard` as well.